youtube_dl/extractor/youtube.py

```diff
@@ -2005,6 +2005,21 @@ def _extract_filesize(media_url):
                     formats.append(a_format)
             else:
                 error_message = extract_unavailable_message()
+                if not error_message:
+                    reason_list = try_get(
+                        player_response,
+                        lambda x: x['playabilityStatus']['errorScreen']['playerErrorMessageRenderer']['subreason']['runs'],
+                        list) or []
+                    for reason in reason_list:
+                        if not isinstance(reason, dict):
+                            continue
+                        reason_text = try_get(reason, lambda x: x['text'], compat_str)
+                        if reason_text:
+                            if not error_message:
```

Comment:
```suggestion
if reason_text and not error_message:
```

Reply:
That wouldn't work at all, those two conditions are separate for a reason, you can't use += to add to None, therefore the first condition sets the message to an empty string so that we can add to it. Your suggestion would also stop working after the first part of the reason, because the message would no longer be falsy.